test_of_the_basic_function.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Test of the basic function
+
+oscap-report < ./tests/test_data/arf-report.xml > report.html
+
+grep -q "xccdf_org\.ssgproject\.content_rule_enable_fips_mode" < report.html
+
+echo "Report generation success"
```

Comment:
It will always pass. You probably have to `exit 1` somewhere or you can add `set -e -o pipefail` at the beginning.